### PR TITLE
GEODE-5014: Remove usage of org.json from geode-assembly module

### PR DESCRIPTION
**Pull request by jdeppe-pivotal**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [x] Does `gradlew build` run cleanly?

- [x] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-assembly/src/test/java/org/apache/geode/test/junit/rules/HttpResponseAssert.java [hunk 1]
```diff
@@ -87,23 +86,17 @@ public HttpResponseAssert statusIsOk() {
...
-  public JSONArray getJsonArray() {
-    JSONTokener tokener = new JSONTokener(responseBody);
-    JSONArray array = new JSONArray(tokener);
-    return array;
-  }
-
-  public ListAssert<Object> hasJsonArray() {
-    JSONTokener tokener = new JSONTokener(responseBody);
-    JSONArray array = new JSONArray(tokener);
+  public ListAssert<Object> hasJsonArrayOfDoubles() throws IOException {
```
Comment (jinmeiliao) [inline geode-assembly/src/test/java/org/apache/geode/test/junit/rules/HttpResponseAssert.java]: If this is named as hasJsonArrayOfDoubles should it return ListAssert<Double>?

Comment (jdeppe-pivotal) [reply geode-assembly/src/test/java/org/apache/geode/test/junit/rules/HttpResponseAssert.java]: Good point. Did you mean it should return `ListAssert<Double>`?

Comment (jinmeiliao) [reply geode-assembly/src/test/java/org/apache/geode/test/junit/rules/HttpResponseAssert.java]: Oh, yeah, of course.

**Comment by jdeppe-pivotal** [comment]
precheckin is green

**Pull request merged**